### DISCUSSION TITLE
"File is not natively supported" on brand new Intel Ultra 7 165U notebook

### DISCUSSION THREAD

**Discussion by de7t1ny** [mifi/lossless-cut#2228] [Support]
Have been using LosslessCut for years on my dell notebook with Intel Core i5 8365U with videos .mp4/h265 that i record with me DJI action cam.
Now I got a new Dell notebook with Intel Ultra 7 165U and LosslessCut tells me that "File is not natively supported".
AFAIK the video codec h265 needs to be supported by the GPU hardware but since its a brand new model i cannot imagine that this is not the case? Perhaps it is so new that I need to wait for ffmpeg to support it?

According to specs h265 is supported 
https://www.intel.com/content/www/us/en/products/sku/237329/intel-core-ultra-7-processor-165u-12m-cache-up-to-4-90-ghz/specifications.html#specs-1-0-4

Disabling/Enabling "Enable HEVC / H265 hardware decoding (you may need to turn this off if you have problems with HEVC files)" does not help.

Thank you very much for explaining me whats going on here...

**Comment by mifi** [comment]
Native H265 support relies on hardware and driver support in the operating system. are you on the latest version of your OS and are drivers installed/updated? also you can try to open the file in google Chrome and see if it plays back there. chrome uses the same engine as losslesscut for playback

**Reply by de7t1ny** [reply]
Thank you very much for your reply. Its a brand new (business) laptop from dell (Latitude 7450) and its on the latest version of Win 11 (24H2 Build 262100.2161).
When I drag&drop a file into (new chrome based) Edge I can only hear sound, no video.
All the preinstalled Dell apps (support assist, command update) tell me, that everything is up to date.
So I installed Intels own drivers "Intel® Arc™ and Iris® Xe Grafics – Windows*".

Besides the fact that the Intel Arc software and driver have 1,6GB very sadly no change.
I guess I cannot do more about "drivers".

![Screenshot 2024-10-31 104527](https://github.com/user-attachments/assets/1714415a-9b45-41a6-a00d-f3a6ede944cf)
![Screenshot 2024-10-31 105726](https://github.com/user-attachments/assets/699813c6-8cf6-4879-95e3-bf9c4a1d3379)

Is ffmpeg is not dealing right with my CPU/GPU yet and they need to update it?

**Reply by mifi** [reply]
If it doesn't work in Chromium/Chrome/Edge either, then I don't know how to solve it unfortunately. Maybe report an issue there. LosslessCut uses Chromium for playback so there's not much I can do if Chromium doesn't support it. You can still use the ffmpeg-assisted playback in losslesscut though. You can also try a different h265 file and see if it applies to all H265 files or just the DJI ones

**Reply by de7t1ny** [reply]
Thank you, so I understand that the problem is not ffmpeg but chromium and I have to wait until they fix this topic.
Please keep on going with your absolutely awesome app! I already purchased it a year ago in MS Store, just to support you (even though of course i am using github version to be more up to date ;-) )

If something changes or I have new information I will post it here.

**Reply by de7t1ny** [reply]
Hi Mifi, got it solved now. My Dell Laptop is the first one that has a new "function".
On first start (usually at factory) it gets service tag and some other options configured.
First time there is also an option for h265/hevc activation/deactivation. 
I don't know whats the reason for this but they deactivated it on my mainboard.
Since Dell wants to have this "hacker proofe" it can only be done on first boot.
Therefore I got a mainboard replacement. When the technician did the unsite support I was able to see the screen and the configuration.
After that VLC realized that gpu hardware supports hevc but not chrome and firefox. Therefore losslesscut behaviour also did not change. 
So I had to install Win11 from scratch, now everything is working.

**Reply by mifi** [reply]
Ok thanks for reporting back in case someone else faces the same issue!
